com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueProcessor.cs

```diff
@@ -32,9 +35,15 @@ internal class RpcQueueProcessor
         /// ProcessReceiveQueue
         /// Public facing interface method to start processing all RPCs in the current inbound frame
         /// </summary>
-        public void ProcessReceiveQueue(NetworkUpdateStage currentStage)
+        internal void ProcessReceiveQueue(NetworkUpdateStage currentStage)
         {
-            bool advanceFrameHistory = false;
+            //In case there is no Singleton, then just exit
+            if(!NetworkManager.Singleton)
+            {
+                return;
+            }
```

Comment:
why? if there is no `NetworkManager.Singleton`, we are probably in trouble already, right?